src/config/kernel.md

```diff
@@ -22,34 +17,14 @@ modules.
 ## Removing old kernels
 
 When updating the kernel, old versions are left behind in case it is necessary
-to roll back to an older version. Over time, many old kernel version accumulate
-and make updating dmks modules take a long time. Thus, it may be advisable to
-clean old kernels from time to time.
+to roll back to an older version. Over time, many old kernel versions can
+accumulate, taking up a lot of space and making dkms modules updates take a long
+time. Thus, it may be advisable to clean old kernels from time to time.
 
 Removing old kernels is done using the
 [vkpurge(8)](https://man.voidlinux.org/vkpurge.8) utility. `vkuprge` comes
-pre-installed on every Void Linux system. This utility runs the necessary hooks
-when removing old kernels.
-
-To list kernels that can be purged:
-
-```
-$ vkpurge list
-3.8.2_1
```

Comment:
I'm okay with the removal of this content, but this is one place I would like a second opinion.

Reply:
About this:

Pros:

* I feel like people should be able to open man pages to see what the comments do, instead of relying on just the docs
* If anything changes in vkpurge, it isn't necessary to change the documentation in several places

Cons:

* A doc that just gives you the command is much more welcoming

Reply:
Yeah I have no strong feelings one way or the other. I think consistency is nice and will help in general. Having this stay is friendly though.

Reply:
i feel the pros listed outweigh the cons, but wouldn't object to the content staying in. Either way, "`vkuprge`" -> "`vkpurge`". :-)

Reply:
fiexd! :p